src/styl/site.styl

```diff
@@ -1,7 +1,5 @@
-  // styl property ordering: https://github.com/rossPatton/stylint/blob/master/src/data/ordering.json
-
-@import 'kouto-swiss'
-@import '../../globie-library/helpers'
+@import '~kouto-swiss/index.styl'
```

Comment:
how does this work?

Reply:
`~` is basically node_modules and other common module paths